tests/e2e/test_sync_app_data.py

```diff
@@ -46,6 +46,7 @@ async def test_fetch_content_and_filter(self):
             block_range=block_range_1,
             config=self.config,
             missing_file_name=missing_files,
+            ipfs_access_key=os.environ["IPFS_ACCESS_KEY"],
```

Comment:
This was missed in a previous PR... e2e tests aren't run in our CI so it was not noticed.